examples/std/.cargo/config.toml

```diff
@@ -0,0 +1,4 @@
+[build]
+target = "x86_64-unknown-linux-gnu"
```

Comment:
Is there a way to specify "host triple"? (this needs to override the `.cargo/config.toml` from the repo root).

Reply:
I'll add a comment.